libs/langgraph/langgraph/pregel/remote.py

```diff
@@ -64,6 +74,15 @@ def __init__(
 
         If `client` or `sync_client` are provided, they will be used instead of the default clients.
```

Comment:
i would also add that if at least one of the 3 (url/client/sync_client) needs to be specified. something similar to how i describe in this how-to guide maybe https://github.com/langchain-ai/langgraph/pull/2218/files